### ISSUE TITLE
Manually generate token

### ISSUE THREAD

**Issue by fernandojsg** [mattupstate/flask-jwt#42]
I'm currently using flask-jwt in my project with angularjs frontend. The users can get the token providing  the user/pass and everything works just fine.

The problem appeared when I needed to get a token to authenticate an external application. I did the following:
```python
@api_bp.route("/external_auth/<id_key>/<json_hash>")
def external_auth(id_key,json_hash):
    
    external_api_key = ExternalApiKey.query.get(id_key)
    if external_api_key == None:
        abort(401)

# blalbalbla specific app code

    return json.dumps({"token":generate_jwt_token({"user_id":data['user_id'], ... extra params})})
```

As I didn't want to use app/password for the external api calls, but I needed the token I basically used your code to generate a token with my custom payload.

```python
from flask import current_app
from itsdangerous import (
    TimedJSONWebSignatureSerializer,
    SignatureExpired,
    BadSignature
)
from datetime import timedelta

def generate_jwt_token(payload):
    expires_in = current_app.config['JWT_EXPIRATION_DELTA']
    if isinstance(expires_in, timedelta):
        expires_in = int(expires_in.total_seconds())
    expires_in_total = expires_in + current_app.config['JWT_LEEWAY']
    serializer = TimedJSONWebSignatureSerializer(
        secret_key=current_app.config['JWT_SECRET_KEY'],
        expires_in=expires_in_total,
        algorithm_name=current_app.config['JWT_ALGORITHM']
    )

    return serializer.dumps(payload).decode('utf-8')
```

My question is... is there any "pretty" way to access your flask_jwt code to do this? I feel like using private functions or objects doesn't looks nice, so that's why I duplicated most of the token generation code.

**Comment by sureshjoshi** [comment]
What about just using 
from flask.ext.jwt import generate_token ?

I use it to send out custom payloads.

**Comment by fernandojsg** [comment]
Hi @sureshjoshi thank you for your answer, but at first I tried to do that:
```from flask_jwt import jwt_required, current_user, generate_token```
but I just realized that my version doesn't include the ```generate_token``` function defined.
I uninstalled it and installed again, but it just keep taking the same code :/ 
It's like if it's downloading the following version of the code: 
https://github.com/mattupstate/flask-jwt/commit/2353630d32417a3e5954b89f2da5c254399d7dba

Before including that function. Do I supose to manually include another repository to install the lastest version? I'm just using the pip installer.
